### ISSUE TITLE
Something wrong with Kwf_Form_Field_File (bug)

### ISSUE THREAD

**Issue by Sogl** [koala-framework/koala-framework#464]
Hi!

Today I found a strange bug in this peace of code:
```php
$tab = $tabs->add();
$tab->setTitle(trl('Личные данные'));

$tab->fields->add(new Kwf_Form_Field_File('Picture', trlKwf('File')))
        ->setShowPreview(true)
        ->setAllowOnlyImages(true);

$fio = new Kwf_Form_Container_FieldSet(trl('ФИО'));
$fio->setWidth(550);

$fio->fields->add(new Kwf_Form_Field_TextField('lastname', trlKwf('Lastname')))
            ->setWidth(400)
            ->setAllowBlank(false);
$fio->fields->add(new Kwf_Form_Field_TextField('firstname', trlKwf('Firstname')))
            ->setAllowBlank(false)
            ->setWidth(400);
$fio->fields->add(new Kwf_Form_Field_TextField('middlename', trl('Отчество')))
        ->setWidth(400)
        ->setAllowBlank(false);

$tab->fields->add($fio);
```

What I see in browser:
![image](https://cloud.githubusercontent.com/assets/7556290/11802294/fc787604-a33a-11e5-977f-d6018ba00e97.png)

Normal view must be:
![image](https://cloud.githubusercontent.com/assets/7556290/11802414/2da4ba8e-a33c-11e5-9eeb-7d66899d7c54.png)

`или переместите сюда` means `or drag here` in English. I searched the whole project and find this line in `trl.xml`:
```xml
<text>
<en>or drag here</en>
<de>oder hier hinziehen</de>
<id>2053</id>
<ru>или переместите сюда</ru>
<ro>or drag here</ro>
</text>
```

Why this happening?

KWF 3.8

**Comment by nsams** [comment]
you can also find it in Kwf_js/Form/File.js

Can you reproduce that but with kwf-app-demo and latest kwf 3.8?

**Comment by Sogl** [comment]
>Can you reproduce that but with kwf-app-demo and latest kwf 3.8?

On production server no error, but exists on test machine...
`clear-cache` and `build` can't help

**Comment by nsams** [comment]
confirmed, it's a regression introduced recently when removing swfupload I guess

**Comment by Ben-Ho** [comment]
This bug is fixed with ceb74abcc88f64d209fd345384c1cd1bd75df89b and then merged: 5828a71068a84379f427563a7676de0579c7c98d
